aw_watcher_window/main.py

```diff
@@ -92,10 +93,11 @@ def main():
                 poll_time=args.poll_time,
                 strategy=args.strategy,
                 exclude_title=args.exclude_title,
+                exclude_titles=[re.compile(re.escape(title), re.IGNORECASE) for title in args.exclude_titles]
```

Comment:
Why are you escaping this? If you are escaping, then it's not really matching a regex, as all regex-special characters (`.[]()+*?` etc) would be escaped and wouldn't work (iirc).

Reply:
I did it this way because it is the way that worked for me in all cases, but it is perfectly possible to remove it

Reply:
What do you mean "all cases"? This breaks regex functionality completely. Perhaps you used a bad regex?

Reply:
It is not so much that they are bad regex, but that by putting part of a title it excludes it, but without escaping the characters it works the same.